src/color/Color.ts

```diff
@@ -186,13 +139,9 @@ export class Color {
    * @return {Color} thisArg
    */
   toGrayscale() {
-    const source = this.getSource(),
-      average = parseInt(
-        (source[0] * 0.3 + source[1] * 0.59 + source[2] * 0.11).toFixed(0),
-        10
-      ),
-      currentAlpha = source[3];
-    this.setSource([average, average, average, currentAlpha]);
+    const [r, g, b, a] = this.getSource(),
+      average = Math.round(r * 0.3 + g * 0.59 + b * 0.11);
+    this.setSource([average, average, average, a]);
     return this;
```

Comment:
this would be a possible change, and we can do that outside the cleanup/refactor scope.
Something that also should be done maybe that is part of cleanup is consolidathing this average code in a function util